lib/book.rb

```diff
@@ -0,0 +1,15 @@
+# The Book class represents a book with a title and an author.
+# It provides getters and setters for the title and author.
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] @kifle23 great work adding a method to add rental in the book class, however in order to achieve the has-many relationship the method needs to include person, date and the book itself.
```suggestion
  def add_rental(person, date)
     Rental.new(date, self, person)
  end
```